plugins/scaffolder-backend-module-rails/src/actions/fetch/rails/index.test.ts

```diff
@@ -90,14 +93,12 @@ describe('fetch:rails', () => {
   });
 
   beforeEach(() => {
-    mockFs({ [`${mockContext.workspacePath}/result`]: {} });
+    mockDir.addContent({
```

Comment:
This needs to be a `setContent` call, otherwise the tests might affect each other

Reply:
replaced setContent with addContent